Unity-Technologies/ScriptableRenderPipeline
# Fixed asset creation causing NullReference exception.

### Purpose of this PR
 Fixes NullReference exception when creating a new render pipeline asset and no render pipeline asset was assigned in Graphics Settings.

---
### Release Notes
 Added

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- Other: 

**Automated Tests**: Added two editor tests to check both render pipeline and forward renderer asset are created without errors.

Any test projects to go with this to help reviewers?

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=lw/fix-asset-creation&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**: Low

---
### Comments to reviewers
 Downsampling type is queried at runtime instead of creation time. Which also makes more sense in case we need to change it at runtime.
